source/content/core-updates.md

```diff
@@ -11,15 +11,19 @@ This doc includes instructions to make core updates to WordPress and Drupal site
 
 Drupal 9 sites on Pantheon use [Integrated Composer](/guides/integrated-composer) to allow one-click core updates through the Dashboard.
 
-To check for available updates, navigate to **Code** in the Dev tab of the site's Dashboard. Click **Check Now**. If updates are available, click **Apply Updates**.
+To check for available updates:
+
+1. Navigate to **Code** in the Dev tab of the site's Dashboard. 
+
+1. Click **Check Now** and then click **Apply Updates** if updates are available.
 
 ## Drupal 8 Composer-Managed Sites
 
-Drupal 8 sites managing core with Composer are not compatible with Pantheon's One-click updates and must update core using Composer exclusively. For instructions, see [Build Tools](/guides/build-tools/update) or [convert the site to Integrated Composer](/guides/composer-convert).
+Drupal 8 sites managing core with Composer are not compatible with Pantheon's One-click updates. You will need to convert to [Drupal 9](/drupal-9) which uses [Integrated Composer](/guides/integrated-composer).
```

Comment:
It is possible to convert to Integrated Composer without upgrading to Drupal 9:

https://pr-6221--pantheon-docs.my.pantheonfrontend.website/docs/guides/composer-convert

The reference to Build Tools is still relevant, as sites that are still on Build Tools can't use one-click updates.